docs/BR.md

```diff
@@ -1810,28 +1810,28 @@ If the CA asserts compliance with these Baseline Requirements, all certificates
 
 #### 7.1.2.1 Root CA Certificate Profile
 
-| __Field__                      | __Description__ |
-| ---                            | ------          |
-| `tbsCertificate`               | |
-| \ \ \ \ `version`              | MUST be v3(2) |
-| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) and less than 2^159^ containing at least 64 bits of output from a CSPRNG. |
-| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
-| \ \ \ \ `issuer`               | Encoded value MUST be byte-for-byte identical to the encoded `subject` |
-| \ \ \ \ `validity`             | See [Section 7.1.2.1.1](#71211-root-ca-validity) |
-| \ \ \ \ `subject`              | See [Section 7.1.2.10.2](#712102-ca-certificate-naming) |
-| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
-| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
-| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
-| \ \ \ \ `extensions`           | See [Section 7.1.2.1.2](#71212-root-ca-extensions) |
-| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
-| `signature`                    | |
+| __Field__                  | __Description__ |
+| ---                        | ------          |
+| `tbsCertificate`           | |
+|     `version`              | MUST be v3(2) |
+|     `serialNumber`         | MUST be a non-sequential number greater than zero (0) and less than 2¹⁵⁹ containing at least 64 bits of output from a CSPRNG. |
+|     `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+|     `issuer`               | Encoded value MUST be byte-for-byte identical to the encoded `subject` |
+|     `validity`             | See [Section 7.1.2.1.1](#71211-root-ca-validity) |
+|     `subject`              | See [Section 7.1.2.10.2](#712102-ca-certificate-naming) |
+|     `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+|     `issuerUniqueID`       | MUST NOT be present |
+|     `subjectUniqueID`      | MUST NOT be present |
+|     `extensions`           | See [Section 7.1.2.1.2](#71212-root-ca-extensions) |
+| `signatureAlgorithm`       | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                | |
 
 ##### 7.1.2.1.1 Root CA Validity
 
 | __Field__   | __Minimum__ | __Maximum__ |
 | -           | ----        | ----        |
 | `notBefore` | One day prior to the time of signing | The time of signing |
-| `notAfter`  | 2922 days (e.g. 8 years)  | 9132 days (e.g. 25 years) |
+| `notAfter`  | 2922 days (approx. 8 years)  | 9132 days (approx. 25 years) |
```

Comment:
Justification for change: 

```Use of "e.g." to mean "approximately" in "2922 days (e.g. 8 years)" is not appropriate. Consider using "approx." instead.```

Comment: Agree. Fully spelling out "approximately" might be helpful for non-native English speakers.